keyboards/keychron/common/wireless/lkbt51.c

```diff
@@ -785,7 +785,7 @@ static void query_rsp_handler(uint8_t* data, uint8_t len) {
 
     switch (data[1]) {
         case LKBT51_CMD_IO_TEST:
-            factory_test_send(data, len);
+            factory_test_send(false, data, len);
```

Comment:
```suggestion
            factory_test_send(true, data, len);
```
it's based on usb raw hid